examples/petsc/ex11p.cpp

```diff
@@ -410,26 +410,18 @@ int main(int argc, char *argv[])
    }
 
    // 12. Free the used memory.
-   if (!use_slepc)
-   {
-      delete lobpcg;
-   }
-   else
-   {
-      delete slepc;
-   }
+   delete lobpcg;
+   delete slepc;
    delete precond;
    delete M;
    delete A;
+   delete pA;
+   delete pM;
 #if defined(MFEM_USE_SUPERLU) || defined(MFEM_USE_STRUMPACK)
    delete Arow;
 #endif
-
    delete fespace;
-   if (order > 0)
-   {
-      delete fec;
-   }
+   delete fec;
```

Comment:
I think we need to verify that the `fed` did not come from `mesh->GetNodes()` otherwise we'll delete it twice.